scripts/update_localization.rb

```diff
@@ -73,6 +76,15 @@
                       "messages#{@options[:pot_suffix]}"
     end
   end
+
+  # Remove trailing space
+  OS.windows?
+    info "Windows detected. Removing trailing space..."
```

Comment:
Trailing spaces should be stripped even on non-windows. Ruby string strip should probably remove the trailing spaces. Did we already have check disallowing trailing spaces for CI?

Reply:
We have, so every time I run update_localization.rb CI fails.

The cause for me to add an OS.windows? is that **every** time I use it on Windows it adds trailing space, but **none** have I experienced the strange behaviour on Linux.

Reply:
If someone manually edits stuff on Linux, they might make a mistake so for that reason I think it would be nice to have the trailing spaces strip on any platform.

Reply:
Then we should add this in check_formatting.rb (?)

Reply:
As you said there's already a check there against this problem, and if the problem only occurs when changing the locales, then I think it's fine to have here. I won't accept any kind of extra mess in check_formatting.rb, it's kind of incidental that some of the checks there actually fix what they detect. If I had a choice in that check_formatting.rb would *not* do any modifications, but due to the tools it needs to call, that is not possible.